cmd/core/main.go

```diff
@@ -138,13 +141,19 @@ func main() {
 		os.Exit(1)
 	}
 
+	capabilities, err := utils.GetEnabledCapabilities(disableCaps)
+	if err != nil {
+		setupLog.Error(err, err.Error())
```

Comment:
```suggestion
		setupLog.Error(err, "unable to get enabled capabilities")
```

两个一样的error message没必要输出2次